gtsam/gtsam.i

```diff
@@ -2759,21 +2759,29 @@ virtual class EssentialMatrixFactor : gtsam::NoiseModelFactor {
 };
 
 #include <gtsam/slam/dataset.h>
+
 class SfmTrack {
-  Point3 point3() const;
+  SfmTrack();
+  SfmTrack(const gtsam::Point3& pt);
```

Comment:
About this new constructor, what happens to the measurements if you use them to triangulate a point? Should the constructor not accept measurements as an argument as well?

Reply:
you mean a vector of measurements as a constructor argument?

Reply:
No, it's acceptable to add them one at a time.